cmd/agent/app/start.go

```diff
@@ -51,27 +48,12 @@ import (
 var (
 	startCmd = &cobra.Command{
 		Use:   "start",
-		Short: "Start the Agent",
-		Long:  `Runs the agent in the foreground`,
+		Short: "(DEPRECATED) Use \"run\" instead to start the Agent",
```

Comment:
Cobra command contain a [deprecated](https://godoc.org/github.com/spf13/cobra#Command.Deprecated) string.
Is it useful in our case ?

Reply:
I think it would.